install-govready-q.sh

```diff
@@ -131,33 +136,44 @@ fi
 
 echo $SPACER
 
+# TODO - Need a test to see if jq is installed on system; commented out code in meantime
+#
 # Use jq and cut to extract hostname from govready-url param in local/environment.json
 # environment_json=$(<local/environment.json)
-govready_url=`cat local/environment.json | jq .\"govready-url\"`
-# echo "govready_url is $govready_url"
-govready_host_and_port=$(echo $govready_url | cut -f2 -d\" | cut -f3 -d/)
-echo "govready_host_and_port is $govready_host_and_port"
-
-
-if [ $NONINTERACTIVE ];
-then
-	echo ""
-	echo "*********************************"
-	echo "* Starting GovReady-Q Server... *"
-	echo "*********************************"
-	echo ""
-	python3 manage.py runserver "$govready_host_and_port"
-else
-	# prompt the user if they want to run the webserver now
-	# this currently runs synchronously, in the foreground, so it needs to be the last
-	# functional line of the script
-	while true;
-	do
-		read -p "Do you want to run the GovReady-Q now, in the foreground? [y/n] " answer
-		case $answer in
-			[Yy]* ) python3 manage.py runserver "$govready_host_and_port"; break;;
-			[Nn]* ) break;;
-			* ) echo "Please answer 'yes' or 'no'";;
-		esac
-	done
-fi
+# govready_url=`cat local/environment.json | jq .\"govready-url\"`
+# # echo "govready_url is $govready_url"
+# govready_host_and_port=$(echo $govready_url | cut -f2 -d\" | cut -f3 -d/)
+# echo "govready_host_and_port is $govready_host_and_port"
+
+echo ""
+echo "***********************************"
+echo "* GovReady-Q Server configured... *"
+echo "***********************************"
+echo ""
+echo "To start GovReady-Q, run: "
+echo "  python3 manage.py runserver"
+echo "or"
+echo "  python manage.py runserver"
+
+
+# TODO - Need a better way to start GovReady because of different use cases
+#        Commenting out code in meantime
+#
+# if [ $NONINTERACTIVE ];
+# then
+# 	echo "Starting GovReady-Q Server..."
+# 	python3 manage.py runserver "$govready_host_and_port"
+# else
+# 	# prompt the user if they want to run the webserver now
+# 	# this currently runs synchronously, in the foreground, so it needs to be the last
+# 	# functional line of the script
+# 	while true;
+# 	do
+# 		read -p "Do you want to run the GovReady-Q now, in the foreground? [y/n] " answer
+# 		case $answer in
+# 			[Yy]* ) python3 manage.py runserver "$govready_host_and_port"; break;;
+# 			[Nn]* ) break;;
+# 			* ) echo "Please answer 'yes' or 'no'";;
+# 		esac
+# 	done
+# fi
```

Comment:
Should probably delete the commented code completely if it's not being used, but other than that, this PR looks great to me.

Reply:
Ok. Makes sense. I'm just afraid I'll forget how something was done...